pyproject.toml

```diff
@@ -7,8 +7,12 @@ requires = [
     "Cython>=0.29.13",  # Note: sync with setup.py
     "numpy==1.13.3; python_version=='3.6' and platform_system!='AIX'",
     "numpy==1.14.5; python_version>='3.7' and platform_system!='AIX'",
+    "numpy==1.15.4; python_version=='3.6' and platform_system!='AIX'",
+    "numpy==1.15.4; python_version=='3.7' and platform_system!='AIX'",
```

Comment:
you can remove those two lines, it's duplicating from above, and was only introduced because on master we dropped support for numpy < 1.15, but we didn't do that on 1.0.x

Reply:
and the previous line changed to == from >=